chaoscenter/graphql/server/pkg/database/mongodb/config/operations.go

```diff
@@ -10,20 +10,20 @@ import (
 )
 
 // CreateConfig creates a new server config with unique key
-func CreateConfig(ctx context.Context, config *ServerConfig) error {
-	err := mongodb.Operator.Create(ctx, mongodb.ServerConfigCollection, config)
+func CreateConfig(ctx context.Context, config *ServerConfig, mongodbOperator mongodb.MongoOperator) error {
```

Comment:
Is it possible to create an interface and receivers here similar to what we have in other places?